README.md

```diff
@@ -7,21 +7,55 @@ open standards:
 * [OAuth 2.0](http://oauth.net/2/)
 * [SCIM v2](http://www.simplecloud.info/) (still in draft phase)
 
-OSIAM is published under the MIT licence, giving you the greatest freedom possible to
-utilize OSIAM in you project or product.
+OSIAM is published under the MIT licence, giving you the greatest freedom
+possible to utilize OSIAM in you project or product.
+
+## Use cases
+
+* Single app with OSIAM as free backing store for users and groups and local
+  auth service. [Learn more](docs/single-app-use-case.md).
```

Comment:
I think "click here" is a little bit different to this style. I like "Lean more". Everyone who is a fan of "Learn more" please "click here"!